addons/phone_validation/models/mail_thread_phone.py

```diff
@@ -219,6 +219,10 @@ def _assert_phone_field(self):
     def _phone_get_sanitize_triggers(self):
         """ Tool method to get all triggers for sanitize """
         res = [self._phone_get_country_field()] if self._phone_get_country_field() else []
+        res.extend(
```

Comment:
Note that I am unsure we should really recompute sanitized number when changing partner country. Issue is that you may potentially trigger a lot of computation just by updating a partner's country. It is not what you expect, if a partner has a lot of leads for example, we should probably let the sanitized phone number as it is.

Maybe we could add partner field, aka when changing the partner, ok to retry a sanitize when updating the partner of a record, seems legit and not too recompute-prone.

Reply:
Indeed I suppose it could get a bit expensive..